src/openrct2/world/MoneyEffect.cpp

```diff
@@ -43,7 +43,7 @@ void MoneyEffect::CreateAt(money32 value, const CoordsXYZ& effectPos, bool verti
     moneyEffect->sprite_height_positive = 30;
     moneyEffect->sprite_identifier = SpriteIdentifier::Misc;
     moneyEffect->MoveTo(effectPos);
-    moneyEffect->type = SPRITE_MISC_MONEY_EFFECT;
+    moneyEffect->type = EnumValue(MiscSpriteType::MoneyEffect);
```

Comment:
Can we declare `type` as a MiscSpriteType instead?

Reply:
Vehicle uses this as Vehicle::Type, thats what I meant by splitting up the variable, Vehicle should have its own type and Misc should have its own type.